examples/ptb/genetxt.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+#encoding: utf-8
+
+""" Sample script of recurrent neural network language model for generating text
+
+This code is ported from following implementation.
+https://github.com/longjie/chainer-char-rnn/blob/master/sample.py
+
+"""
+import argparse
+import codecs
+
+from chainer import cuda, Variable
```

Comment:
Don't import class `Variable`. And, move this line before `import chainer.functions`